pypeit/scripts/ql_keck_deimos.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+"""
+This script runs PypeIt *fast* on a set of DEIMOS images
+"""
+import os
+import glob
+import copy
+import yaml
+import io
+import numpy as np
+import argparse
+
+from pypeit.scripts import scriptbase
+
+from pypeit.scripts.utils import Utilities
+from pypeit.par import PypeItPar
+from pypeit.par.util import parse_pypeit_file
+from pypeit.pypeitsetup import PypeItSetup
+from pypeit import msgs
+from pypeit.scripts import run_pypeit
+from pypeit.par.util import make_pypeit_file
+from pypeit import utils
+
+from IPython import embed
+
+class QLKeckDEIMOS(scriptbase.ScriptBase):
+
+    @classmethod
+    def name(cls):
+        """
+        Return the name of the executable.
+        """
+        return 'pypeit_ql_keck_deimos'
+
+    @classmethod
+    def get_parser(cls, width=None):
+        import argparse
+        parser = super().get_parser(description='Pypeit QL for Keck/DEIMOS',
+                                    width=width, formatter=argparse.RawDescriptionHelpFormatter)
+        parser = argparse.ArgumentParser(description='Script to run PypeIt in QuickLook on a set of '
+                                                    'Keck/DEIMOS files',
+                                        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+        parser.add_argument('full_rawpath', type=str, help='Full path to the raw files')
+        parser.add_argument('--redux_path', type=str, help='Path to where reduction products lie')
```

Comment:
Maybe it would be better not to leave the default of `--redux_path` as None, but to set it to be the current working directory, like is done in `pypeit_setup`.

Reply:
I've mimicked `run_pypeit` here.
`None` seems ok.